src/transforms/rename.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+""" 
+    Rename(Dict(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ))
+
+Tha transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+""" 
+struct Rename3 <: Stateless
+    names::Dict{Symbol,Symbol}
+end
+
+function apply(transform::Rename3, table)
+    new_table = (;)
+
+    for col_name in Tables.columnnames(table)
+        col_value = Tables.getcolumn(table, col_name)
```

Comment:
Try to follow the code style in the other transforms. We avoid variable names with underscore for example. We use 2 whitespaces for indentation, etc.